mii/grpc_related/proto/modelresponse_pb2_grpc.py

```diff
@@ -76,63 +76,64 @@ class ModelResponseServicer(object):
     """Missing associated documentation comment in .proto file."""
     def Terminate(self, request, context):
         """Missing associated documentation comment in .proto file."""
+        ERROR_MSG = 'Method not implemented!'
         context.set_code(grpc.StatusCode.UNIMPLEMENTED)
-        context.set_details('Method not implemented!')
-        raise NotImplementedError('Method not implemented!')
+        context.set_details()
```

Comment:
```suggestion
        context.set_details(ERROR_MSG)
```

Reply:
Thanks for the review. It was a  copy-paste error.